### PR TITLE
Fix typo and improve context

### PR DESCRIPTION
**Pull request by dakeshi**
- fix some typo
- resolved alignment issue on crowdin
- some improved description

### CODE REVIEW

File: English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md [hunk 1]
```diff
@@ -59,183 +59,307 @@ Transfer, vote, issuance of token, or participation in token offering. Sending s
...
+14.1 Interface statement
+rpc GetAssetIssueList (EmptyMessage) returns (AssetIssueList) {};  
+14.2 Node
+Fullnode and soliditynode.  
+14.3 Parameters
+EmptyMessage: null.  
+14.4 Returns
+AssetIssueList: AssetIssueContract list containing information on all issued tokens.  
+14.5 Function
+Query of list of all issued tokens. Display of all issued tokens for user’s reference.
```
Comment (dakeshi) [inline English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md]: To make it clear, add `query of` in the beginning

File: English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md [hunk 2]
```diff
@@ -59,183 +59,307 @@ Transfer, vote, issuance of token, or participation in token offering. Sending s
...
+17.1 Interface statement
+rpc GetAssetIssueListByTimestamp (NumberMessage) returns (AssetIssueList){};  
+17.2 Node
+Soliditynode.  
+17.3 Parameters
+NumberMessage: current timestamp (the number of milliseconds since 1970).  
+17.4 Returns
+AssetIssueList: AssetIssueContract list including detailed information of the tokens.
+17.5 Function
+Query of list of issued tokens by timestamList of issued tokens by timestamp. Display of current nodes for users’ reference.
```
Comment (dakeshi) [inline English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md]: To make it clear, add `query of` in the description.

Comment (kookiekrak) [reply English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md]: I think this line is wrong, should be:

Query list of issued tokens by timestamp. Display of current nodes for users’ reference.

File: English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md (hunk 1 above)
Comment (kookiekrak) [reply English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md]: Should be:

Query list of all issued tokens.

Comment (dakeshi) [reply English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md]: Thanks @kookiekrak How was `14. Query of list of all tokens`? Do it need to be fixed too?

File: English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md (hunk 2 above)
Comment (dakeshi) [reply English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md]: it looks better

**Comment by kookiekrak** [comment]
@dakeshi ty for your help!

**Pull request merged**